conda.recipe/lux-widget/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "lux-widget" %}
```

Comment:
Should we put this file inside the lux-widget repo?

Reply:
Yeah I can do that